go.mod

```diff
@@ -48,34 +48,36 @@ require (
 	go.opentelemetry.io/otel/exporters/zipkin v1.11.2
 	go.opentelemetry.io/otel/sdk v1.11.2
 	go.opentelemetry.io/otel/trace v1.11.2
-	go.temporal.io/api v1.15.0
-	go.temporal.io/sdk v1.20.0
+	go.temporal.io/api v1.18.1
+	go.temporal.io/sdk v1.21.1
 	go.uber.org/automaxprocs v1.5.1
 	go.uber.org/ratelimit v0.2.0
-	golang.org/x/exp v0.0.0-20230213192124-5e25df0256eb
+	golang.org/x/exp v0.0.0-20230303215020-44a13b063f3e
 	golang.org/x/net v0.7.0
 	golang.org/x/sync v0.1.0
-	google.golang.org/genproto v0.0.0-20230209215440-0dfe4f8abfcc
+	google.golang.org/genproto v0.0.0-20230223222841-637eb2293923
 	google.golang.org/grpc v1.53.0
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.26.1
-	k8s.io/apiextensions-apiserver v0.26.1
-	k8s.io/apimachinery v0.26.1
+	k8s.io/api v0.26.2
+	k8s.io/apiextensions-apiserver v0.26.2
+	k8s.io/apimachinery v0.26.2
 	k8s.io/cli-runtime v0.26.1
-	k8s.io/client-go v0.26.1
-	k8s.io/code-generator v0.26.1
+	k8s.io/client-go v0.26.2
+	k8s.io/code-generator v0.26.2
 	k8s.io/klog v1.0.0
 	k8s.io/metrics v0.26.1
 	sigs.k8s.io/controller-runtime v0.14.1
 )
 
+require github.com/redis/go-redis/v9 v9.0.2 // indirect
```

Comment:
Please merge in group below

Reply:
As always the go tooling made this mess... sigh. Fixed manually.